build/DependencyVersions.props

```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.3.0-preview2-4095</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25331-02</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
```

Comment:
I got confirmation from @Petermarcu that the version that was just picked up by core-setup is the correct one: https://github.com/dotnet/core-setup/commit/1768bf1d4c7fd317d4a99d6c0cfba851ff2a61cd, so it is 25405-01 as I am using here.